app/services/edition_filter.rb

```diff
@@ -9,20 +9,23 @@ class EditionFilter
   SORT_KEYS = %w[last_updated].freeze
   DEFAULT_SORT = "-last_updated"
 
-  attr_reader :filters, :sort, :page, :per_page
+  attr_reader :filters, :sort, :page, :per_page, :user
 
-  def initialize(params)
+  def initialize(user, **params)
     @filters = params[:filters].to_h.symbolize_keys
     @sort = allowed_sort?(params[:sort]) ? params[:sort] : DEFAULT_SORT
     @page = params.fetch(:page, 1).to_i
     @per_page = params[:per_page]
+    @user = user
   end
 
   def editions
     revision_joins = { revision: %i[content_revision tags_revision] }
     scope = Edition.where(current: true)
+                   .joins("LEFT OUTER JOIN access_limits ON access_limits.id = access_limit_id")
```

Comment:
I think we can use `.left_outer_joins(:access_limit)` or `.left_joins(:access_limit)` rather than writing the whole join.

Reply:
Phew - I couldn't find this when I looked initially 🎆 .